controllers/errors.js

```diff
@@ -1,8 +1,25 @@
-exports.handle404s = (req, res, next) => {
-  res.status(404).send({ msg: "Cannot find what you wanted" });
+exports.handleCustomErrors = (req, res, next) => {
+  res
+    .status(404)
+    .send({ msg: "You tried to look for something and it wasn't found" });
+};
+
+exports.handle400s = (err, req, res, next) => {
+  if (err.status === 400) {
+    res.status(400).send({ msg: "Bad request" });
+  } else {
+    next(err);
+  }
 };
 
 exports.handle500s = (err, req, res, next) => {
-  console.log(err);
-  res.status(500).send({ msg: "Error in code somewhere" });
+  if (err.status === 500) {
+    res.status(500).send({ msg: "Error in code somewhere" });
+  } else {
```

Comment:
you are doubling up on functions for your 500 error handler, you have this function written here and also a callback written in app - you only need one or the other